featuretools/primitives/standard/aggregation_primitives/__init__.py

```diff
@@ -0,0 +1,57 @@
+# flake8: noqa
+from featuretools.primitives.standard.aggregation_primitives.all import All
+from featuretools.primitives.standard.aggregation_primitives.any import Any
+from featuretools.primitives.standard.aggregation_primitives.avg_time_between import (
+    AvgTimeBetween,
+)
+from featuretools.primitives.standard.aggregation_primitives.count import Count
+from featuretools.primitives.standard.aggregation_primitives.count_above_mean import (
+    CountAboveMean,
+)
+from featuretools.primitives.standard.aggregation_primitives.count_below_mean import (
+    CountBelowMean,
+)
+from featuretools.primitives.standard.aggregation_primitives.count_greater_than import (
+    CountGreaterThan,
+)
+from featuretools.primitives.standard.aggregation_primitives.count_inside_nth_std import (
+    CountInsideNthSTD,
+)
+from featuretools.primitives.standard.aggregation_primitives.count_inside_range import (
+    CountInsideRange,
+)
+from featuretools.primitives.standard.aggregation_primitives.count_less_than import (
+    CountLessThan,
+)
+from featuretools.primitives.standard.aggregation_primitives.count_outside_nth_std import (
+    CountOutsideNthSTD,
+)
+from featuretools.primitives.standard.aggregation_primitives.count_outside_range import (
+    CountOutsideRange,
+)
+from featuretools.primitives.standard.aggregation_primitives.entropy import Entropy
+from featuretools.primitives.standard.aggregation_primitives.first import First
+from featuretools.primitives.standard.aggregation_primitives.last import Last
+from featuretools.primitives.standard.aggregation_primitives.max import Max
+from featuretools.primitives.standard.aggregation_primitives.mean import Mean
+from featuretools.primitives.standard.aggregation_primitives.median import Median
+from featuretools.primitives.standard.aggregation_primitives.min import Min
+from featuretools.primitives.standard.aggregation_primitives.mode import Mode
+from featuretools.primitives.standard.aggregation_primitives.n_most_common import (
+    NMostCommon,
+)
+from featuretools.primitives.standard.aggregation_primitives.num_true import NumTrue
+from featuretools.primitives.standard.aggregation_primitives.num_unique import NumUnique
+from featuretools.primitives.standard.aggregation_primitives.percent_true import (
+    PercentTrue,
+)
+from featuretools.primitives.standard.aggregation_primitives.skew import Skew
+from featuretools.primitives.standard.aggregation_primitives.std import Std
+from featuretools.primitives.standard.aggregation_primitives.sum import Sum
+from featuretools.primitives.standard.aggregation_primitives.time_since_first import (
+    TimeSinceFirst,
+)
+from featuretools.primitives.standard.aggregation_primitives.time_since_last import (
+    TimeSinceLast,
+)
+from featuretools.primitives.standard.aggregation_primitives.trend import Trend
```

Comment:
These can all be relative imports

Reply:
We have a pre-commit hook that converts them all to absolute.

Reply:
shucks, no worries.